src/mode/modeHandler.ts

```diff
@@ -79,6 +79,14 @@ export class ModeHandler implements vscode.Disposable {
   public syncCursors() {
     setImmediate(() => {
       if (this.vimState.editor) {
+        let cursors: Range[] = [];
+        for (const { start, end } of this.vimState.editor.selections) {
```

Comment:
Instead of looping + pushing, could you write this by `map`ping over `this.vimState.editor.selections`?